bindgen-tests/tests/expectations/tests/issue-1554.rs

```diff
@@ -1,6 +1,9 @@
 #![allow(dead_code, non_snake_case, non_camel_case_types, non_upper_case_globals)]
 #![cfg(feature = "nightly")]
 #![feature(non_exhaustive)]
+pub type Planet_ctype = ::std::os::raw::c_uint;
+pub const Planet_earth: Planet_ctype = 0;
+pub const Planet_mars: Planet_ctype = 1;
```

Comment:
So the PR is generating the constants for (some of) the existing Rust `enum` styles -- not sure if that is what others were expecting or not.

Reply:
Oh, so are you saying that a non exhaustive rustified enum shouldn't generate constants? I'm happy to only do it if either `try_from_raw` or `from_raw_unchecked` is enabled.